lms/security.py

```diff
@@ -173,6 +181,14 @@ def _get_user(request):
 
 
 def includeme(config):
-    config.set_security_policy(SecurityPolicy())
+    config.set_security_policy(
+        SecurityPolicy(
+            path_policy_mapping={
+                "/admin": LMSGoogleSecurityPolicy,
+                "/googleauth": LMSGoogleSecurityPolicy,
```

Comment:
I wondered with the non-ordered nature of dicts could cause the order of the mappings here to get randomised and potentially cause the results of security policy lookups to be random. But since Python 3.7 [dicts are ordered now](https://mail.python.org/pipermail/python-dev/2017-December/151283.html) (it says so in the official Python docs as well)

Reply:
Yeah ordered since 3.6 as an implementation detail, guaranteed as part of python in 3.7.

Added a couple of comments about it.